docs/source/docs/log.md

```diff
@@ -0,0 +1,59 @@
+# Logging
+
+## DeepReg logging
+
+DeepReg uses `logging` to log the messages. The log level is controlled by the
```

Comment:
... uses the Python module `logging` to log....

Reply:
updated